src/components/slider/slider.scss

```diff
@@ -21,16 +22,16 @@
 }
 
 .mdc-slider {
-    @include mdc-slider-highlight-color(
-        var(--lime-primary-color, $mdc-theme-primary)
-    );
-    @include mdc-slider-thumb-color(
-        var(--lime-primary-color, $mdc-theme-primary)
-    );
-    @include mdc-slider-value-pin-fill-color(
-        var(--lime-primary-color, $mdc-theme-primary)
-    );
-    @include mdc-slider-value-pin-ink-color(on-primary);
+    // @include mdc-slider-highlight-color(
```

Comment:
We have 2 issues here:

1. The knob is not on the slider bar.
2. We should may be remove the shadow from .mdc-slider__thumb on line 207? .
We don't see the value inside the knob, this might not be an issue but I should mention it here since it is a different feature. The slider value can be seen on top of the knob when draging the knob

<img width="441" alt="Screenshot 2021-07-15 at 09 59 04" src="https://user-images.githubusercontent.com/22968497/125755138-7a16f518-2a75-4f30-96e1-f42d7c98d332.png">